### DISCUSSION TITLE
Modern Polars - 4  Reshaping and Tidy Data

### DISCUSSION THREAD

**Discussion by giscus** [kevinheavey/modern-polars#21] [General]
# Modern Polars - 4  Reshaping and Tidy Data

A side-by-side comparison of the Polars and Pandas libraries.

https://kevinheavey.github.io/modern-polars/tidy.html

**Comment by howsiyu** [comment]
We can remove some `to_pandas` since seaborn v0.13.0.

**Reply by kevinheavey** [reply]
thanks, updated
